Dashboard/app/js/lib/components/devices/AssignmentsEditView/index.jsx

```diff
@@ -20,6 +20,8 @@ export default class AssignmentsEditView extends React.Component {
   // lifecycle methods
   componentDidMount() {
     this.validateAssignmentName(this.props.inputValues.assignmentName);
+    this.validateDate('startDateMsg', this.props.inputValues.startDate || '');
+    this.validateDate('expireDateMsg', this.props.inputValues.toDate || '');
```

Comment:
This causes the red text to show up next to the dates as soon as you click on _Create new assignment_